CHANGELOG.md

```diff
@@ -17,6 +17,10 @@
 * Added the `RouterDelegate.router(_:initialManeuverBufferWhenReroutingFrom:)` method and related `NavigationServiceDelegate` and `NavigationViewControllerDelegate` methods to configure rerouting buffering. ([#3454](https://github.com/mapbox/mapbox-navigation-ios/pull/3454))
 * Added `RouterDelegate.router(_:requestBehaviorForReroutingWith:)` and related `NavigationServiceDelegate` and `NavigationViewControllerDelegate` methods to provide customization for rerouting mechanism by providing a user-defined route as a reroute. ([#3472](https://github.com/mapbox/mapbox-navigation-ios/pull/3472))
 
+### Location tracking
+
+* Fixed an issue where the transition from Active to Passive navigation didn't update underlying Navigator properly. It is a programmer error to have more than one active RouteController simultaneously. ([#3652](https://github.com/mapbox/mapbox-navigation-ios/pull/3652))
```

Comment:
Should we have this disclaimer here about simultaneous instances? 
And it sounds a bit strange to me. Maybe this is better?
```suggestion
* Fixed an issue where the transition from Active to Passive navigation didn't update underlying Navigator properly. It is an integration issue to have more than one active RouteController simultaneously. ([#3652](https://github.com/mapbox/mapbox-navigation-ios/pull/3652))
```

Reply:
> And it sounds a bit strange to me. Maybe this is better?

This is a commonly used phrase. For example, take a look on this Apple documentation: https://developer.apple.com/documentation/watchconnectivity/wcsessionactivationstate/wcsessionactivationstateinactive?changes=_2_6&language=objc

Reply:
> Should we have this disclaimer here about simultaneous instances?

Isn't "It is a programmer error to have more than one active RouteController simultaneously." enough?

Reply:
Nevermind, I'm okay with this text, just didn't get the reason why dow we mention it here.

Reply:
I have a follow up PR that will add a documentation entry related to this requirement. If this change is accepted at all.